sklearn/linear_model/_ridge.py

```diff
@@ -262,7 +261,7 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
         assumed to be specific to the targets. Hence they must correspond in
         number.
 
-    sample_weight : float or numpy array of shape (n_samples,), default=None
+    sample_weight : float or array-like of shape (n_samples,), default=None
         Individual weights for each sample. If sample_weight is not None and
         solver='auto', the solver will be set to 'cholesky'.
```

Comment:
docstring should say what sample_weight as float means. It's not "individual weights" for each sample.

Reply:
I hope you approve my short explanation on floats.